modules/tinymce/src/core/main/ts/commands/IndentOutdent.ts

```diff
@@ -77,7 +77,11 @@ const handle = (editor: Editor, command: string): void => {
   });
 };
 
+const indent = (editor: Editor) => handle(editor, 'indent');
+const outdent = (editor: Editor) => handle(editor, 'outdent');
```

Comment:
```suggestion
const indent = (editor: Editor): void => handle(editor, 'indent');
const outdent = (editor: Editor): void => handle(editor, 'outdent');
```

Reply:
I added void return types here but there is probably lots and lots of places if we need to add them to functions that gets called within a block scope without a return statement. Is the idea to always have explicit type annotations that would be very verbose but then again this language is very verbose anyway when it comes to types.